dspace2hydra.rb

```diff
@@ -122,28 +123,40 @@ def item_log_path(bag, started_at)
   # Process all of the bags individually
   bags.each do |bag|
     item_id = "ITEM@#{bag.item.item_id}"
-    begin
-      bag_start = DateTime.now
-      start_logging_to(item_log_path(bag, started_at), item_id: item_id)
-      @logger.info('Started')
-      server = HydraEndpoint::Server.new(CONFIG['hydra_endpoint'], type_config, started_at)
-
-      # We've decided that if a work has 2+ files, then it should be a Parent work with each file being a
-      # child.
-      if bag.files_for_upload.count > 1
-        work = Work::MigrationStrategy::ParentWithChildren.new(bag, server, CONFIG, type_config)
-      else
-        work = Work::MigrationStrategy::SingleWork.new(bag, server, CONFIG, type_config)
-      end
-      work.process_bag
 
-      @logger.info("Finished in #{time_since(bag_start)}")
-    rescue StandardError => e
-      @logger.fatal("#{e.message} : #{e.backtrace.join("\n\t")}")
-    ensure
-      stop_logging_to(item_log_path(bag, started_at), item_id: item_id)
+    if CONFIG['extract_files']
+      bag.files_for_upload.each do |f|
+        @logger.info(f.metadata_full_path)
+        begin
+          FileUtils.copy f.full_path, File.join(CONFIG['extract_files'] + '/' + bag.item.item_id + '_' + f.name)
```

Comment:
This is the only thing that jumped out at me, you want File.join to take care of whether or not to add the slash incase the argument was entered with/without it..

`FileUtils.copy f.full_path, File.join(CONFIG['extract_files'], "#{bag.item.item_id}_#{f.name})`